spec/requests/admins_spec.rb

```diff
@@ -2,23 +2,46 @@
 
 RSpec.describe '/admins controllers' do
   describe "#add_to_admins, remove_from_admins" do
-    context 'patch add_to_admins' do
-      before(:each) do
-        post '/users/sign_in', :params => {
-          "email" => "asdfdf@mail.com",
-          "password"=> "secret",
-          "password_confirmation"=> "secret"
-        }
+    context 'add_to_admins and remove from admins' do
+
+      before do
+        post '/users', :params => { "user": {
+          "email": "dfddfdfddsfd@mail.com",
+          "password": "secret",
+          "password_confirmation": "secret",
+          "worker_attributes": {
+            "first_name": "dffdfdfdfdfd",
+            "last_name": "Bradi",
+            "age": 30,
+            "role": "Manager" } } }
+        delete '/users/sign_out'
+
+        post '/users', :params => { "user": {
+          "email": "fddsda@mail.com",
+          "password": "secret",
+          "password_confirmation": "secret",
+          "worker_attributes": {
+            "first_name": "sdfaw",
+            "last_name": "Bradi",
+            "age": 30,
+            "role": "Manager" } } }
+
+        user = User.last
+        user.is_admin = true
+        user.save
       end
 
-      it 'should return 200 http status code' do
-        patch '/add_admin/2'
+      it 'should return 200 http status code if user become admin' do
+        patch '/add_admin/1'
```

Comment:
do not use hardcoded `id` values
should look like
`let(:user) { create(:user) }`

``` it 'should blablabla do
          patch "/add_admin/#{user.id}"
          ...
      end
```

Reply:
expected: {"message"=>"dfddfdfddsfd@mail.com is admin"}
     got: {"errors"=>"already admin!"}
I really don't understand why it change id for current user, not for target